src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -385,11 +385,20 @@ Copyright (C) Microsoft Corporation. All rights reserved.
   <PropertyGroup>
     <_GenerateBindingRedirectsIntermediateAppConfig>$(IntermediateOutputPath)$(TargetFileName).config</_GenerateBindingRedirectsIntermediateAppConfig>
   </PropertyGroup>
+  
+  <PropertyGroup Condition ="'$(MSBuildCopyMarkerName)' == ''">
```

Comment:
nit: There's a space before but not after `=`. The convention in this file seems to be no spaces.